CodeSignal/redux-rags
# Update minor/patch versions, run npm audit fix

Closes #16.

To test:
- Check out this branch
- Run `npm install` and/or `npm audit`
- Don't see >9000 security warnings due to `lodash`
